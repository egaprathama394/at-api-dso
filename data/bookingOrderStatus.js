export default {
  orderDaimoCode: 'DFA202000000123',
  pssOrderStatus: '2',
};
