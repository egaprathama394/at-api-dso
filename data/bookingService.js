const randomstring = require('randomstring');

const fullNameMax = randomstring.generate({
  charset: 'string',
  length: '110',
});

const noPhoneMax = randomstring.generate({
  charset: 'numeric',
  length: '17',
});

const noPhoneRandom = randomstring.generate({
  charset: 'numeric',
  length: '18',
});

const fullNameRandom = randomstring.generate({
  charset: 'string',
  length: '111',
});

const randomNopol1 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol2 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol3 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

const randomNopol4 = randomstring.generate({
  charset: 'numeric',
  length: '4',
});

export default {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: '',
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data submit booking service',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: '',
  policeRegNo: `AB${randomNopol1}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for submit booking service max lenght

export const maxInput = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: `${fullNameMax}`,
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data booking service dengan inputan max char',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: `081${noPhoneMax}`,
  policeRegNo: `AC${randomNopol2}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for submit booking service max lenght + 1

export const negativeRequest = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: `${fullNameRandom}`,
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data submit booking service dengan inputan max + 1 char',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: `081${noPhoneRandom}`,
  policeRegNo: `AD${randomNopol3}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

// data for cancel booking service

export const inputDataForCancel = {
  address: '',
  bookingDateTime: '2020-04-01 10:00:00',
  branchCode: 'D004',
  city: 'DKI JAKARTA',
  fullName: '',
  jobService: [
    {
      actual: 0.0,
      id: 0,
      isActive: false,
      isPickedJob: false,
      jobCode: '10000',
      jobDescription: 'CHECKING 10.000',
      price: 0,
      rate: 0,
    },
  ],
  notes: 'Data untuk cancel booking service',
  orderEndTime: '2020-04-01 15:00:00',
  orderStartTime: '2020-04-01 10:00:00',
  phoneNo: '',
  policeRegNo: `AA${randomNopol4}XYZ`,
  vehicleType: 'LUXIO',
  vin: '',
  year: 2014,
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const cancelRequest = {
  orderNo: '',
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

// data for display booking service

export const displayBookingService = {
  isWalkIn: false,
  orderBy: '',
  AccountType: '1',
  Action: '',
  ApplicationName: '8EA6E777-AA05-4CAF-9F54-5B6A106760E2',
  DeviceType: '0',
  OperatingSystem: '0',
  UserVersion: '1.0.0',
};

export const invalidDisplayBookingService = {
  isWalkIn: '',
  orderBy: '',
  AccountType: '',
  Action: '',
  ApplicationName: '',
  DeviceType: '',
  OperatingSystem: '',
  UserVersion: '',
};
