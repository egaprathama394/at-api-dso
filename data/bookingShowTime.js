export default {
  branchCode: 'D004',
  scheduleDate: '2020-09-30 00:00:00',
  vehicleType: 'XENIA MT',
  jobs: [
    {
      jobCode: 'OTHER',
      jobDescription: 'PERBAIKAN LAINNYA',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const negativeRequest = {
  branchCode: 'D004',
  scheduleDate: '2018-10-26 00:00:00',
  vehicleType: 'XENIA MT',
  jobs: [
    {
      jobCode: 'OTHER',
      jobDescription: 'PERBAIKAN LAINNYA',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};
