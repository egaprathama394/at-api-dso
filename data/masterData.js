
export default {
  companyCode: '0003',
  masterDataInfo: [
    {
      lastSync: '1900-01-01 00:00:00',
      objectName: 'Branch',
    },
  ],
  vehicleBrandCode: 'DAIHATSU',
  accountType: '1',
  action: '',
  deviceType: '0',
  operatingSystem: '0',
  serVersion: '1.0.0',
};

export const JobServiceCompany = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'JobService_Company',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const VehicleBrand = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'vehicleBrand',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const district = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'district',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const province = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'province',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const region = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'region',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

export const holidayCalendar = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'holidayCalendar',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',

};

// looping data for job service company

export const loopVehicleType = [
  {
    vehicleType: 'AYLA',
    title: 'test case get master data job service company where vehicle = AYLA',
  },
  {
    vehicleType: 'GRANMAX',
    title: 'test case get master data job service company where vehicle = GRANMAX',
  },
  {
    vehicleType: 'GRANMAX1.3',
    title: 'test case get master data job service company where vehicle = GRANMAX1.3',
  },
  {
    vehicleType: 'GRANMAX1.5',
    title: 'test case get master data job service company where vehicle = GRANMAX1.5',
  },
  {
    vehicleType: 'LUXIO',
    title: 'test case get master data job service company where vehicle = LUXIO',
  },
  {
    vehicleType: 'Sigra 1.0',
    title: 'test case get master data job service company where vehicle = Sigra 1.0',
  },
  {
    vehicleType: 'Sigra 1.2',
    title: 'test case get master data job service company where vehicle = Sigra 1.2',
  },
  {
    vehicleType: 'SIRION',
    title: 'test case get master data job service company where vehicle = SIRION',
  },
  {
    vehicleType: 'TERIOS',
    title: 'test case get master data job service company where vehicle = TERIOS',
  },
  {
    vehicleType: 'XENIA1.0',
    title: 'test case get master data job service company where vehicle = XENIA1.0',
  },
  {
    vehicleType: 'XENIA1.3',
    title: 'test case get master data job service company where vehicle = XENIA1.3',
  },
  {
    vehicleType: 'OTHERS',
    title: 'test case get master data job service company where vehicle = OTHERS',
  },
];

export const VehicleTestDrive = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'VehicleTestDrive',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};

export const parameterConfigApp = {
  vehicleBrandCode: 'DAIHATSU',
  companyCode: '0003',
  masterDataInfo: [
    {
      objectName: 'ParameterConfigApplication',
      lastSync: '1900-06-17 00:00:00',
    },
  ],
  action: '',
  userVersion: '1.0.0',
  deviceType: '0',
  operatingSystem: '0',
  accountType: '1',
};
