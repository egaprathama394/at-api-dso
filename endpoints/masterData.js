/* eslint no-unused-vars: off */
/* global env */
const supertest = require('supertest');
const env = require('dotenv').config();

process.env.NODE_TLS_REJECT_UNAUTHORIZED = 0;

const api = supertest(process.env.API_BASE_URL_MOBILE);

export default body => api.post('/api/masterdatainfo')
  .set('ClientTag', process.env.CLIENT_TAG)
  .set('server-code', process.env.SERVER_CODE_MOBILE)
  .send(body);
