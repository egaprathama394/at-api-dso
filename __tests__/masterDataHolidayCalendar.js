/* eslint-disable max-len */
/* eslint-disable eqeqeq */
/* eslint-disable no-shadow */
import getMasterData from '../endpoints/masterData';
import * as masterDataSchema from '../schemaValidator/masterData';
import * as body from '../data/masterData';

describe('Master Data Holiday Calendar', () => {
  test('Can get master data Holiday Calendar successfully - positive test cases', async () => {
    const response = await getMasterData(body.holidayCalendar);
    expect(response.status).toEqual(200);
    expect(response.body).toMatchSchema(masterDataSchema.holidayCalendarSchema);
    expect(response.body.holidayCalendar.length).toEqual(132);
    expect(response.body).toMatchSnapshot();
  });
});
