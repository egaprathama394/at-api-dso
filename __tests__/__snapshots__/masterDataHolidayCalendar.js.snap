// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Master Data Holiday Calendar Can get master data Holiday Calendar successfully - positive test cases 1`] = `
Object {
  "acknowledge": 1,
  "branch": Array [],
  "build": "0",
  "district": Array [],
  "hobbies": Array [],
  "holidayCalendar": Array [
    Object {
      "holidayDate": "2016-01-01 00:00:00",
      "holidayDescription": "Tahun Baru Masehi",
      "id": 1,
    },
    Object {
      "holidayDate": "2016-01-02 00:00:00",
      "holidayDescription": "Cuti Massal",
      "id": 2,
    },
    Object {
      "holidayDate": "2016-02-08 00:00:00",
      "holidayDescription": "Tahun Baru Imlek 2567 Kongzili",
      "id": 3,
    },
    Object {
      "holidayDate": "2016-03-09 00:00:00",
      "holidayDescription": "Hari Raya Nyepi Tahun Baru Saka 1938",
      "id": 4,
    },
    Object {
      "holidayDate": "2016-03-25 00:00:00",
      "holidayDescription": "Wafat Isa Almasih",
      "id": 5,
    },
    Object {
      "holidayDate": "2016-05-01 00:00:00",
      "holidayDescription": "Peringatan Hari Buruh Nasional",
      "id": 6,
    },
    Object {
      "holidayDate": "2016-05-05 00:00:00",
      "holidayDescription": "Kenaikan Isa Almasih",
      "id": 7,
    },
    Object {
      "holidayDate": "2016-05-06 00:00:00",
      "holidayDescription": "Isra Miraj Nabi Muhammad SAW",
      "id": 8,
    },
    Object {
      "holidayDate": "2016-05-22 00:00:00",
      "holidayDescription": "Hari Raya Waisak 2560",
      "id": 9,
    },
    Object {
      "holidayDate": "2016-07-04 00:00:00",
      "holidayDescription": "Cuti Bersama",
      "id": 10,
    },
    Object {
      "holidayDate": "2016-07-05 00:00:00",
      "holidayDescription": "Cuti Bersama",
      "id": 11,
    },
    Object {
      "holidayDate": "2016-07-06 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1437 H",
      "id": 12,
    },
    Object {
      "holidayDate": "2016-07-07 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1437 H",
      "id": 13,
    },
    Object {
      "holidayDate": "2016-07-08 00:00:00",
      "holidayDescription": "Cuti Bersama",
      "id": 14,
    },
    Object {
      "holidayDate": "2016-07-09 00:00:00",
      "holidayDescription": "Cuti Massal",
      "id": 15,
    },
    Object {
      "holidayDate": "2016-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 16,
    },
    Object {
      "holidayDate": "2016-09-12 00:00:00",
      "holidayDescription": "Hari Raya Idul Adha 1437 H",
      "id": 17,
    },
    Object {
      "holidayDate": "2016-10-02 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1438 H",
      "id": 18,
    },
    Object {
      "holidayDate": "2016-12-12 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 19,
    },
    Object {
      "holidayDate": "2016-12-25 00:00:00",
      "holidayDescription": "Hari Raya Natal",
      "id": 20,
    },
    Object {
      "holidayDate": "2016-12-26 00:00:00",
      "holidayDescription": "Cuti Bersama",
      "id": 21,
    },
    Object {
      "holidayDate": "2017-01-01 00:00:00",
      "holidayDescription": "Tahun Baru Masehi 2017",
      "id": 22,
    },
    Object {
      "holidayDate": "2017-01-02 00:00:00",
      "holidayDescription": "Cuti Massal Tahun Baru Masehi 2017",
      "id": 23,
    },
    Object {
      "holidayDate": "2017-01-28 00:00:00",
      "holidayDescription": "Tahun Baru Imlek 2566 Kongzili",
      "id": 24,
    },
    Object {
      "holidayDate": "2017-03-28 00:00:00",
      "holidayDescription": "Hari Raya Nyepi Th Saka 1939",
      "id": 25,
    },
    Object {
      "holidayDate": "2017-04-14 00:00:00",
      "holidayDescription": "Wafat Isa Almasih",
      "id": 26,
    },
    Object {
      "holidayDate": "2017-04-24 00:00:00",
      "holidayDescription": "Isra Miraj Nabi Muhammad SAW",
      "id": 27,
    },
    Object {
      "holidayDate": "2017-05-01 00:00:00",
      "holidayDescription": "Peringatan Hari Buruh Internasional",
      "id": 28,
    },
    Object {
      "holidayDate": "2017-05-11 00:00:00",
      "holidayDescription": "Hari Raya Waisak",
      "id": 29,
    },
    Object {
      "holidayDate": "2017-05-25 00:00:00",
      "holidayDescription": "Kenaikan Isa Al-Masih",
      "id": 30,
    },
    Object {
      "holidayDate": "2017-07-01 00:00:00",
      "holidayDescription": "Hari Lahir Pancasila",
      "id": 31,
    },
    Object {
      "holidayDate": "2017-06-24 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 32,
    },
    Object {
      "holidayDate": "2017-06-25 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1438 H",
      "id": 33,
    },
    Object {
      "holidayDate": "2017-06-26 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1438 H",
      "id": 34,
    },
    Object {
      "holidayDate": "2017-06-27 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 35,
    },
    Object {
      "holidayDate": "2017-06-28 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 36,
    },
    Object {
      "holidayDate": "2017-06-29 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 37,
    },
    Object {
      "holidayDate": "2017-06-30 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 38,
    },
    Object {
      "holidayDate": "2017-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 39,
    },
    Object {
      "holidayDate": "2017-09-01 00:00:00",
      "holidayDescription": "Rari Raya Idul Adha 1438 H",
      "id": 40,
    },
    Object {
      "holidayDate": "2017-09-21 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1439 H",
      "id": 41,
    },
    Object {
      "holidayDate": "2017-12-01 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 42,
    },
    Object {
      "holidayDate": "2017-12-25 00:00:00",
      "holidayDescription": "Hari Raya Natal",
      "id": 43,
    },
    Object {
      "holidayDate": "2017-12-26 00:00:00",
      "holidayDescription": "Cuti Massal Hari Raya Natal",
      "id": 44,
    },
    Object {
      "holidayDate": "2017-07-01 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1438 H",
      "id": 53,
    },
    Object {
      "holidayDate": "2018-01-01 00:00:00",
      "holidayDescription": "Tahun Baru Masehi 2018",
      "id": 54,
    },
    Object {
      "holidayDate": "2018-02-16 00:00:00",
      "holidayDescription": "Tahun Baru Imlek 2569 Kongzili",
      "id": 55,
    },
    Object {
      "holidayDate": "2018-03-17 00:00:00",
      "holidayDescription": "Hari Raya Nyepi Th Saka 1940",
      "id": 56,
    },
    Object {
      "holidayDate": "2018-03-30 00:00:00",
      "holidayDescription": "Wafat Isa Almasih",
      "id": 57,
    },
    Object {
      "holidayDate": "2018-04-14 00:00:00",
      "holidayDescription": "Isra Mikraj Nabi Muhammad SAW",
      "id": 58,
    },
    Object {
      "holidayDate": "2018-05-01 00:00:00",
      "holidayDescription": "Peringatan Hari Buruh Internasional",
      "id": 59,
    },
    Object {
      "holidayDate": "2018-05-10 00:00:00",
      "holidayDescription": "Kenaikan Isa Al Masih",
      "id": 60,
    },
    Object {
      "holidayDate": "2018-05-29 00:00:00",
      "holidayDescription": "Hari Raya Waisak 2566",
      "id": 61,
    },
    Object {
      "holidayDate": "2018-06-01 00:00:00",
      "holidayDescription": "Hari Lahir Pancasila",
      "id": 62,
    },
    Object {
      "holidayDate": "2018-06-15 00:00:00",
      "holidayDescription": "Hari RayaIdutl Fitri 1439 H",
      "id": 63,
    },
    Object {
      "holidayDate": "2018-06-16 00:00:00",
      "holidayDescription": "Hari RayaIdutl Fitri 1439 H",
      "id": 64,
    },
    Object {
      "holidayDate": "2018-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 65,
    },
    Object {
      "holidayDate": "2018-08-22 00:00:00",
      "holidayDescription": "Hari Raya Idul Adha 1439 H",
      "id": 66,
    },
    Object {
      "holidayDate": "2018-09-11 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1440 H",
      "id": 67,
    },
    Object {
      "holidayDate": "2018-11-20 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 68,
    },
    Object {
      "holidayDate": "2018-12-25 00:00:00",
      "holidayDescription": "Hari Raya Natal",
      "id": 69,
    },
    Object {
      "holidayDate": "2018-11-01 00:00:00",
      "holidayDescription": "testing libur",
      "id": 71,
    },
    Object {
      "holidayDate": "2018-12-12 00:00:00",
      "holidayDescription": "Testing Libur",
      "id": 72,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru Masehi 2019",
      "id": 73,
    },
    Object {
      "holidayDate": "2019-02-05 00:00:00",
      "holidayDescription": "Tahun Baru Imlek 2570 Kongzili",
      "id": 74,
    },
    Object {
      "holidayDate": "2019-03-07 00:00:00",
      "holidayDescription": "Hari Raya Nyepi Th Saka 1941",
      "id": 75,
    },
    Object {
      "holidayDate": "2019-04-03 00:00:00",
      "holidayDescription": "Isra Mikraj Nabi Muhammad SAW",
      "id": 76,
    },
    Object {
      "holidayDate": "2019-04-17 00:00:00",
      "holidayDescription": "Pemungutan Suara Pemiihan Legislatif dan Presiden Indonesia 2019",
      "id": 77,
    },
    Object {
      "holidayDate": "2019-04-19 00:00:00",
      "holidayDescription": "Wafat Isa Almasih",
      "id": 78,
    },
    Object {
      "holidayDate": "2019-05-01 00:00:00",
      "holidayDescription": "Peringatan Hari Buruh Internasional",
      "id": 79,
    },
    Object {
      "holidayDate": "2019-05-19 00:00:00",
      "holidayDescription": "Hari Raya Waisak 2563",
      "id": 80,
    },
    Object {
      "holidayDate": "2019-05-30 00:00:00",
      "holidayDescription": "Kenaikan Isa Al Masih",
      "id": 81,
    },
    Object {
      "holidayDate": "2019-06-01 00:00:00",
      "holidayDescription": "Hari Lahir Pancasila",
      "id": 82,
    },
    Object {
      "holidayDate": "2019-06-03 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri 1440 H",
      "id": 83,
    },
    Object {
      "holidayDate": "2019-06-04 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri 1440 H",
      "id": 84,
    },
    Object {
      "holidayDate": "2019-06-05 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1440 H",
      "id": 85,
    },
    Object {
      "holidayDate": "2019-06-06 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1440 H",
      "id": 86,
    },
    Object {
      "holidayDate": "2019-06-07 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri 1440 H",
      "id": 87,
    },
    Object {
      "holidayDate": "2019-08-11 00:00:00",
      "holidayDescription": "Hari Raya Idul Adha 1440 H",
      "id": 88,
    },
    Object {
      "holidayDate": "2019-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 89,
    },
    Object {
      "holidayDate": "2019-09-01 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1441 H",
      "id": 90,
    },
    Object {
      "holidayDate": "2019-11-09 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 91,
    },
    Object {
      "holidayDate": "2019-12-24 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Natal",
      "id": 92,
    },
    Object {
      "holidayDate": "2019-12-25 00:00:00",
      "holidayDescription": "Hari Raya Natal",
      "id": 93,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "TESTING TAHUN BARU",
      "id": 94,
    },
    Object {
      "holidayDate": "2019-01-19 00:00:00",
      "holidayDescription": "Testing Libur",
      "id": 106,
    },
    Object {
      "holidayDate": "2019-06-01 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 107,
    },
    Object {
      "holidayDate": "2019-06-02 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 108,
    },
    Object {
      "holidayDate": "2019-06-03 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 109,
    },
    Object {
      "holidayDate": "2019-06-04 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 110,
    },
    Object {
      "holidayDate": "2019-06-05 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1440 H",
      "id": 111,
    },
    Object {
      "holidayDate": "2019-06-06 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1440 H",
      "id": 112,
    },
    Object {
      "holidayDate": "2019-06-07 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 113,
    },
    Object {
      "holidayDate": "2019-06-08 00:00:00",
      "holidayDescription": "Cuti Massal Idul Fitri 1440 H",
      "id": 114,
    },
    Object {
      "holidayDate": "2019-08-11 00:00:00",
      "holidayDescription": "Hari Raya Idul Adha 1440 H",
      "id": 115,
    },
    Object {
      "holidayDate": "2019-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 116,
    },
    Object {
      "holidayDate": "2019-09-01 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1441 H",
      "id": 117,
    },
    Object {
      "holidayDate": "2019-11-09 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 118,
    },
    Object {
      "holidayDate": "2019-12-24 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Natal",
      "id": 119,
    },
    Object {
      "holidayDate": "2019-09-28 00:00:00",
      "holidayDescription": "Holiday Testing",
      "id": 120,
    },
    Object {
      "holidayDate": "2019-09-29 00:00:00",
      "holidayDescription": "Holiday Testing",
      "id": 121,
    },
    Object {
      "holidayDate": "2019-10-05 00:00:00",
      "holidayDescription": "Holiday Testing",
      "id": 122,
    },
    Object {
      "holidayDate": "2020-01-01 00:00:00",
      "holidayDescription": "Tahun Baru 2020",
      "id": 123,
    },
    Object {
      "holidayDate": "2020-01-25 00:00:00",
      "holidayDescription": "Tahun Baru Imlek 2571 Kongzili",
      "id": 124,
    },
    Object {
      "holidayDate": "2020-03-22 00:00:00",
      "holidayDescription": "Isra Miraj Nabi Muhammad SAW",
      "id": 125,
    },
    Object {
      "holidayDate": "2020-03-25 00:00:00",
      "holidayDescription": "Hari Suci Nyepi Tahun Baru Saka 1942",
      "id": 126,
    },
    Object {
      "holidayDate": "2020-04-10 00:00:00",
      "holidayDescription": "Wafat Isa Al Masih",
      "id": 127,
    },
    Object {
      "holidayDate": "2020-05-01 00:00:00",
      "holidayDescription": "Hari Buruh Internasional",
      "id": 128,
    },
    Object {
      "holidayDate": "2020-05-07 00:00:00",
      "holidayDescription": "Hari Raya Waisak 2564",
      "id": 129,
    },
    Object {
      "holidayDate": "2020-05-21 00:00:00",
      "holidayDescription": "Kenaikan Isa Al Masih",
      "id": 130,
    },
    Object {
      "holidayDate": "2020-05-22 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri",
      "id": 131,
    },
    Object {
      "holidayDate": "2020-05-24 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1441 Hijriyah",
      "id": 132,
    },
    Object {
      "holidayDate": "2020-05-25 00:00:00",
      "holidayDescription": "Hari Raya Idul Fitri 1441 Hijriyah",
      "id": 133,
    },
    Object {
      "holidayDate": "2020-05-26 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri",
      "id": 134,
    },
    Object {
      "holidayDate": "2020-05-27 00:00:00",
      "holidayDescription": "Cuti Bersama Hari Raya Idul Fitri",
      "id": 135,
    },
    Object {
      "holidayDate": "2020-06-01 00:00:00",
      "holidayDescription": "Hari Lahir Pancasila",
      "id": 136,
    },
    Object {
      "holidayDate": "2020-07-31 00:00:00",
      "holidayDescription": "Hari Raya Idul Adha 1441 Hijriyah",
      "id": 137,
    },
    Object {
      "holidayDate": "2020-08-17 00:00:00",
      "holidayDescription": "Hari Kemerdekaan RI",
      "id": 138,
    },
    Object {
      "holidayDate": "2020-08-20 00:00:00",
      "holidayDescription": "Tahun Baru Islam 1442 H",
      "id": 139,
    },
    Object {
      "holidayDate": "2020-10-29 00:00:00",
      "holidayDescription": "Maulid Nabi Muhammad SAW",
      "id": 140,
    },
    Object {
      "holidayDate": "2020-12-25 00:00:00",
      "holidayDescription": "Hari Raya Natal",
      "id": 141,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "New year",
      "id": 95,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 96,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 97,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 98,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru2",
      "id": 99,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 100,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 101,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 102,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 103,
    },
    Object {
      "holidayDate": "2019-01-01 00:00:00",
      "holidayDescription": "Tahun Baru",
      "id": 104,
    },
    Object {
      "holidayDate": "2019-01-02 00:00:00",
      "holidayDescription": "test",
      "id": 105,
    },
  ],
  "jobService": Array [],
  "jobServiceCompany": Array [],
  "location": Array [],
  "parameterConfigApplication": Array [],
  "province": Array [],
  "region": Array [],
  "vehicleBrand": Array [],
  "vehicleTestDrive": Array [],
  "version": "1.0",
}
`;
